src/sign/types.ts

```diff
@@ -126,6 +139,21 @@ export const proposalTypes = {
   ]
 };
 
+export const updateProposalTypes = {
+  updateProposal: [
```

Comment:
All other types start with a capital letter, maybe better to make it `UpdateProposal`?